src/fhir/utilities.ts

```diff
@@ -284,4 +284,224 @@ export class FhirUtilities {
       });
     });
   }
+
+  static async populateDB() {
+    const db = Globals.database;
+
+
+    // define schemas
+    const medicationCollection = await db.collection('medication-requirements'
+    // , {
```

Comment:
Are these comments to show the structure?
If so maybe a comment stating that would be good. Otherwise it should be removed.